caravel/models.py

```diff
@@ -444,24 +444,6 @@ def grains(self):
             if self.sqlalchemy_uri.startswith(db_type):
                 return grains
 
-    def dttm_converter(self, dttm):
-        """Returns a string that the database flavor understands as a date"""
-        default = "'{}'".format(dttm.strftime('%Y-%m-%d %H:%M:%S.%f'))
-        iso = dttm.isoformat()
-        d = {
```

Comment:
These per-database defaults is what made things work magically before. I feel like these must still be taken into account or this PR will create a regression on the databases that don't use the global default